### PR TITLE
feat: Added possibility to show more or less filters

### PR DESCRIPTION
**Pull request by MauritsioRK**
https://alice.its.cern.ch/jira/browse/O2B-267

### CODE REVIEW

**Comment by mvegter** [comment]
@MauritsioRK the following is not working propertly:

- Press 'more filters'
- Select 'maintenance'
- Press 'less filters'
- Press 'more filters'
- Expected maintenance to still be selected

File: lib/public/components/Filters/index.js [hunk 1]
```diff
@@ -38,16 +39,18 @@ const checkboxFilter = (model, tags) => {
...
-            }, 'Meer opties'),
-            ...checkboxes.slice(FILTERS_LIMITS),
-        ])
-        : h('.form-group', checkboxes);
+    if (checkboxes.length <= FILTERS_LIMIT) {
+        return checkboxes;
+    } else {
+        const toggleFilters = h('button.btn.btn-primary.mv1#toggleMoreFilters', {
+            onclick: () => model.logs.toggleMoreFilters(),
+        }, ...model.logs.moreFilters ? [iconMinus(), ' Less filters'] : [iconPlus(), ' More filters']);
```
Comment (john-ghatas) [inline lib/public/components/Filters/index.js:47]: Define a getter for the moreFilters var in the model

**Comment by john-ghatas** [comment]
Also rebase with master to get the latest changes :)

**Comment by mvegter** [comment]
I'm going to either disagree with the change requested from @john-ghatas or the implementation of @MauritsioRK . The naming convention for a boolean getter or function returning a boolean should be prefixed with `is` (with the alternatives of `has`, `can` and `should`).

**Comment by MauritsioRK** [comment]
@mvegter I do agree with @john-ghatas that we should use getter methods for this. I have changed the relevant method names accordingly.

**Pull request merged**